en/docs/learn/authentication-session-persistence.md

```diff
@@ -64,51 +64,51 @@ into either the service provider or the WSO2 Identity Server.
             However, if you have persistence, you can rely on it as well. This
             increases the reliability of the overall system.
 
-In WSO2 Identity Server has multiple local caching usage in different
+WSO2 Identity Server has multiple local caching usage in different
 layers to improve the product performance. But most of the caches are
 ready to persist in the database if it is required.
 
 There are three type of data objects that are persisted in the database.
 
 #### 1. Session Data
 
-Once the user get authenticated over WSO2 Identity Server, it will
-create a Session data object which is stored the authenticated user and
-the other authentication flow details. This will store in the database
-to share across the cluster nodes.
+Once the user gets authenticated over WSO2 Identity Server, it will
```

Comment:
Once the user is authenticated, WSO2 Identity Server creates a session data object that stores ..

Reply:
Changed